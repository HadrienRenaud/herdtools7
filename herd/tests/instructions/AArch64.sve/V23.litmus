AArch64 V23
{
 uint64_t 0:X0;
 uint64_t 0:X1;
}
  P0       ;
RDVL X0,#0 ;
RDVL X1,#1 ;

forall 0:X0=0 /\ 0:X1=16
(* Notice: 0:X1 value depends on the target test runs on *)
