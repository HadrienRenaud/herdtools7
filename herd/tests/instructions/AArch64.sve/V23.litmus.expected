Test V23 Required
States 1
0:X0=0; 0:X1=16;
Ok
Witnesses
Positive: 1 Negative: 0
Flag Scalable-Vector-Extensions-is-work-in-progress
Condition forall (0:X0=0 /\ 0:X1=16)
Observation V23 Always 1 0
Hash=2e5dfca61b0bd045b16659236f6c3297

